db/db.go

```diff
@@ -92,6 +92,17 @@ func (d DB) DeletePost(id float32) error {
 	return nil
 }
 
+func (d DB) EditPost(p Post) error {
```

Comment:
Technically, this should be called "UpdatePost", as it's updating the post. And that's the SQL verb used as well.